pkg/datapath/loader/loader.go

```diff
@@ -132,6 +137,63 @@ func nullifyStringSubstitutions(strings map[string]string) map[string]string {
 	return nullStrings
 }
 
+// patchHostNetdevHeaderFile creates a header file corresponding to each new
+// device to which bpf_host is attached, including the second host device and
+// native devices.
```

Comment:
Should this comment include the motivation behind the creation of multiple headers?